src/Kdyby/Doctrine/DI/OrmExtension.php

```diff
@@ -39,7 +39,7 @@ class OrmExtension extends Nette\DI\CompilerExtension
 	 * @var array
 	 */
 	public $managerDefaults = array(
-		'metadataCache' => 'default',
+		'metadataCache' => null,
 		'queryCache' => 'default',
```

Comment:
I'd probably change the defaults to 'filesystem' for the others (in both debug and production mode). What do you think?